<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账单管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,20">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,3,1,1">
     <item>
      <widget class="QLabel" name="label_nowTime">
       <property name="text">
        <string>当前时间:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nowTime">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_setting">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ex">
       <property name="text">
        <string>导出账单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_main">
     <property name="title">
      <string>操作面板</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_timeID">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="rowCount">
           <number>200</number>
          </property>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>项目id</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>项目类别</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>项目实体id</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>开始时间</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>结束时间</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>金额</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_timeID">
          <item>
           <widget class="QLabel" name="label_timeID">
            <property name="text">
             <string>请先输入实体项目编号:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_timeID"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_timeIDadd">
            <property name="text">
             <string>添加计时项目</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_timeIDstop">
            <property name="text">
             <string>结束计时项目</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_timeIDdel">
            <property name="text">
             <string>删除计时项目</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>商品添加</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2,2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>商品类别:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_c">
             <item>
              <property name="text">
               <string>溜冰</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>饮料</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>食物</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>杂项</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>商品价格:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_p"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_addItem">
             <property name="text">
              <string>添加商品</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
